src/main/java/com/mozu/jobs/scheduler/JobScheduler.java

```diff
@@ -700,15 +701,28 @@ private Trigger createCronTrigger(String hours, String minutes, String timezone,
                 + ((siteId != null) ? "_" + siteId.toString() : "");
 
         TimeZone tz = TimeZone.getTimeZone(timezone);
-
+        
+        Integer[] randomNumbers=getRandomNumber();
+        minutes=String.valueOf(Integer.parseInt(minutes )+randomNumbers[0]);
+        String seconds=randomNumbers[1].toString();
+        
         // Create a cron trigger
         Trigger trigger = TriggerBuilder
                 .newTrigger()
                 .withIdentity(identity + TRIGGER, tenantIdString)
                 .withSchedule(
-                        CronScheduleBuilder.cronSchedule("0 " + minutes + " " + hours + " * * ?")
+                        CronScheduleBuilder.cronSchedule( seconds+ " " + minutes + " " + hours + " * * ?")
                                 .inTimeZone(tz)).usingJobData(TENANT_ID_KEY, tenantId)
                 .usingJobData(SITE_ID_KEY, siteId).usingJobData(JOB_NAME, identity).build();
         return trigger;
     }
+    
```

Comment:
It seems weird to returning an array of Integers here.  Just return the Random number and let the method calling it figure out what to do.  Otherwise, you're returning a random array w/o any structure.  I have to remember "is the minute the first or second item in the array."  Hashtable could work, but really calculating the Random number is all this should do.